Source/Core/VideoBackends/Software/SWmain.cpp

```diff
@@ -165,31 +157,17 @@ void VideoSoftware::Shutdown()
 	// Do our OSD callbacks
 	OSD::DoCallbacks(OSD::OSD_SHUTDOWN);
 
-	GLInterface->Shutdown();
-	delete GLInterface;
-	GLInterface = nullptr;
+	delete SWOGLWindow::s_instance;
+	SWOGLWindow::s_instance = nullptr;
```

Comment:
There should be a function within SWOGLWindow that does this (ShudownInstance or something)